packages/mui-base/src/SliderUnstyled/SliderUnstyled.js

```diff
@@ -782,7 +785,9 @@ const SliderUnstyled = React.forwardRef(function SliderUnstyled(props, ref) {
                     // So that VoiceOver's focus indicator matches the thumb's dimensions
                     width: '100%',
                     height: '100%',
+                    ...inputProps.style,
                   }}
+                  {...inputProps}
```

Comment:
`ownerState` should be passed in (similarly to Root, Rail, etc.)

Reply:
@michaldudak I'm not sure to understand why a custom prop `ownerState` should be passed to an HTML element of type `input`. This is not a valid DOM attribute.

Contrary to `Rail`, for example, which is a "styled" component by default: https://github.com/mui-org/material-ui/blob/master/packages/mui-material/src/Slider/Slider.js#L112.

Furthermore I found that if you pass a custom component to Root, Rail etc like a `span`: 
```jsx
components={{ Rail: props => <span {...props} /> }}
```
React will throw a warning on dev runtime because `ownerState` is not a valid HTML attribute.

Reply:
That's why we need a check if a hidden input is indeed a HTML (host) element or a custom component provided via the `components.Input` prop. See how it's done for Thumb in line 751.

As for why - you may provide a custom component in place of the default `<input>` and customize it externally. These customizations may need to know the state of the Slider. 
I realize that customizing the hidden input is not a common use case, but since we allow it, we should be consistent with the API we expose.

Reply:
Alright, I just pushed an update to add the `ownerState` prop :+1: